response.cc

```diff
@@ -0,0 +1,59 @@
+#include "request_handler.h"
+#include "response.h"
+
+
+void Response::SetStatus(const ResponseCode response_code)
+{
+	status = response_code;
+}
+
+
+void Response::SetBody(const std::string& body)
+{
+	response_body = body;
+}
+
+void Response::AddHeader(const std::string& header_name, const std::string& header_value)
+{
+  //std::string head = build_headers();
+  std::pair<std::string, std::string> header(header_name, header_value);
+  headers_container.push_back(header);
+
+}
+
+std::string Response::ToString()
+ {
+  std::string response_header = "";
+  //fix status portion
+  response_header += req->HTTP + " " +  getTextForEnum(status) + crlf;
+  for (unsigned int i = 0; i < headers_container.size(); ++i)
+  {
+    //Header& h = headers[i];
+    response_header += headers_container[i].first;
+    response_header += name_value_separator;
+    response_header += headers_container[i].second;
+    response_header += crlf;
+  }
+
+  response_header += crlf;
+  std::cout << response_header << std::endl;
+  return response_header; 
+
+ }
+
+ //helper function
+
+ std::string Response::getTextForEnum( int enumVal )
+ {
+  switch( enumVal )
+  {
+  case ResponseCode::OK:
+    return "OK";
+  case ResponseCode::INTERNAL_SERVER_ERROR:
+    return "INTERNAL_SERVER_ERROR";
+
+  default:
+    return "Not recognized..";
+  }
```

Comment:
Change text to appropriate HTTP Response Code texts. Here's the list: http://www.restapitutorial.com/httpstatuscodes.html